kubric/simulator.py

```diff
@@ -26,7 +26,8 @@
 @dataclass
 class Object3D:
     sim_filename: str
-    vis_filename: str
+    # TODO: this filename is loaded by pybullet, and can be fetched by getVisualShapeData?
+    # vis_filename: str
```

Comment:
In our case yes, but in general we might want to be able to specify a separate file (e.g. a `.blend`file).

But I was thinking that we should maybe add a constructor (classmethod) `from_urdf(filename)` that sets the vis_filename automatically.

Reply:
Sounds good, for now I will leave it empty "None".
PyBullet exposes this information during export.
We can make it pybullet independent later on?

Reply:
Feel free to open an issue to track this change.
For now, I extract the knowledge out of getVisualShapeData
Otherwise I would have to write a URDF parser now (which pybullet already has)